libpromises/evalfunction.c

```diff
@@ -8607,7 +8573,7 @@ const FnCallType CF_FNCALL_TYPES[] =
                   FNCALL_OPTION_NONE, FNCALL_CATEGORY_COMM, SYNTAX_STATUS_NORMAL),
     FnCallTypeNew("datastate", CF_DATA_TYPE_CONTAINER, DATASTATE_ARGS, &FnCallDatastate, "Construct a container of the variable and class state",
                   FNCALL_OPTION_NONE, FNCALL_CATEGORY_UTILS, SYNTAX_STATUS_NORMAL),
-    FnCallTypeNew("difference", CF_DATA_TYPE_STRING_LIST, SETOP_ARGS, &FnCallSetop, "Returns all the unique elements of list arg1 that are not in list arg2",
+    FnCallTypeNew("difference", CF_DATA_TYPE_STRING_LIST, SETOP_ARGS, &FnCallSetop, "Returns all the unique elements of collected list or array or data container arg1 that are not in collected list or array or data container arg2",
```

Comment:
I think that using the word "collected" here doesn't really help to understand what it does. In fact, now that most functions *are* collecting, I've been wondering if we should refrain from using that term at all. As a user, the "collecting" behavior is something I would expect from functions intuitively, so I don't need it specified. What do you think?

Of course, the reference to lists, arrays and containers should stay.

This goes for all the references below as well.